khelo/lib/ui/flow/matches/match_detail/components/match_detail_highlight_view.dart

```diff
@@ -163,15 +177,27 @@ class MatchDetailHighlightView extends ConsumerWidget {
   }
 
   void showFilterOptionSelectionSheet(
-      BuildContext context, Function(HighlightFilterOption) onTap) {
-    showActionBottomSheet(
+    BuildContext context, {
+    required Function(HighlightFilterOption) onTap,
+    required HighlightFilterOption highlightFilterOption,
+  }) async {
+    return await showActionBottomSheet(
         context: context,
         items: HighlightFilterOption.values
-            .map((e) => BottomSheetAction(
-                  title: e.getString(context),
+            .map((option) => BottomSheetAction(
+                  title: option.getString(context),
+                  child: highlightFilterOption == option
+                      ? SvgPicture.asset(
+                          Assets.images.icCheck,
+                          colorFilter: ColorFilter.mode(
+                            context.colorScheme.primary,
+                            BlendMode.srcATop,
+                          ),
+                        )
```

Comment:
can we create a function that return `Widget?`(optional widget) and pass `boolean flag` to return Widget Or null as this is repeating twice?